Assets/MixedRealityToolkit.Tests/PlayModeTests/FocusProviderTests.cs

```diff
@@ -96,8 +96,6 @@ public IEnumerator TestGazeProviderTargetNotNull()
         [UnityTest]
         public IEnumerator TestOverrideFocusDetails()
         {
-            PlayModeTestUtilities.Setup();
```

Comment:
extra?

Reply:
Yep - if you notice in Setup, this is already called. I found that with the changes in this PR, having this double initialization led to issues, so just opted to remove the double initialization.